app/src/main/java/com/eveningoutpost/dexdrip/BluetoothScan.java

```diff
@@ -83,21 +81,8 @@ protected void onCreate(Bundle savedInstanceState) {
                 Toast.makeText(this, "The android version of this device is not compatible with Bluetooth Low Energy", Toast.LENGTH_LONG).show();
             }
         }
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-            if (ContextCompat.checkSelfPermission(this,
-                    android.Manifest.permission.ACCESS_COARSE_LOCATION)
-                    != PackageManager.PERMISSION_GRANTED) {
-
-                ActivityCompat.requestPermissions(this,
-                        new String[]{android.Manifest.permission.ACCESS_COARSE_LOCATION},
-                        0);
-
-
-            }
-        }
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
-            initializeScannerCallback();
```

Comment:
`initializeScannerCallback()` is no longer needed?
Especially as the changes just add something just for M but take away this for Lollipop?